### ISSUE TITLE
Publish tox4j artifacts somewhere

### ISSUE THREAD

**Issue by robinlinden** [evilcorpltd/aTox#730]
Right now (since bintray was shut down) only the from-source builds are supported. These artifacts should probably be provided using either jitpack, github artifacts, or moving them to Maven Central.

**Comment by roihershberg** [comment]
Maybe we should update the status of this in the README? That those who have GitHub account can access the artifact by clicking the `Actions` tab, the `ci` workflow, choosing one of the green ones and clicking `tox4j` at the bottom of the page.

**Comment by robinlinden** [comment]
Yeah, you're right, we should. :P I added it to my todo when setting that up, but I haven't gotten to it yet.

**Comment by toxifoxy** [comment]
can u please give me the link to the newest repositories or give us a demo to build it after cloning the project from github
like where i lunch the command of building ?how to install dependencies?etc...
Could not determine the dependencies of task ':domain:preBuild'.

Could not create task ':domain:fixPaths'.
Could not resolve all files for configuration ':domain:needFixing'.
> Could not find org.toktok:tox4j-c_x86_64-linux-android:0.2.18.
Searched in the following locations:
- file:/home/toxi/.m2/repository/org/toktok/tox4j-c_x86_64-linux-android/0.2.18/tox4j-c_x86_64-linux-android-0.2.18.pom
- https://dl.google.com/dl/android/maven2/org/toktok/tox4j-c_x86_64-linux-android/0.2.18/tox4j-c_x86_64-linux-android-0.2.18.pom
- https://repo.maven.apache.org/maven2/org/toktok/tox4j-c_x86_64-linux-android/0.2.18/tox4j-c_x86_64-linux-android-0.2.18.pom
Required by:
project :domain
> Could not find org.toktok:tox4j-c_i686-linux-android:0.2.18.
Searched in the following locations:
- file:/home/toxi/.m2/repository/org/toktok/tox4j-c_i686-linux-android/0.2.18/tox4j-c_i686-linux-android-0.2.18.pom
- https://dl.google.com/dl/android/maven2/org/toktok/tox4j-c_i686-linux-android/0.2.18/tox4j-c_i686-linux-android-0.2.18.pom
- https://repo.maven.apache.org/maven2/org/toktok/tox4j-c_i686-linux-android/0.2.18/tox4j-c_i686-linux-android-0.2.18.pom
Required by:
project :domain
> Could not find org.toktok:tox4j-c_armv7a-linux-androideabi:0.2.18.
Searched in the following locations:
- file:/home/toxi/.m2/repository/org/toktok/tox4j-c_armv7a-linux-androideabi/0.2.18/tox4j-c_armv7a-linux-androideabi-0.2.18.pom
- https://dl.google.com/dl/android/maven2/org/toktok/tox4j-c_armv7a-linux-androideabi/0.2.18/tox4j-c_armv7a-linux-androideabi-0.2.18.pom
- https://repo.maven.apache.org/maven2/org/toktok/tox4j-c_armv7a-linux-androideabi/0.2.18/tox4j-c_armv7a-linux-androideabi-0.2.18.pom
Required by:
project :domain
> Could not find org.toktok:tox4j-c_aarch64-linux-android:0.2.18.
Searched in the following locations:
- file:/home/toxi/.m2/repository/org/toktok/tox4j-c_aarch64-linux-android/0.2.18/tox4j-c_aarch64-linux-android-0.2.18.pom
- https://dl.google.com/dl/android/maven2/org/toktok/tox4j-c_aarch64-linux-android/0.2.18/tox4j-c_aarch64-linux-android-0.2.18.pom
- https://repo.maven.apache.org/maven2/org/toktok/tox4j-c_aarch64-linux-android/0.2.18/tox4j-c_aarch64-linux-android-0.2.18.pom
Required by:
project :domain
